README.md

```diff
@@ -214,97 +196,25 @@ If you want to try the example Java benchmarks, there is a separate
-    'wallclock_times': {        # A dict object containing timing results
-        'bmark:VM:variant': [   # A list of lists of in-process iteration times
-            [ ... ], ...        # One list per process execution
-        ]
-    },
-    'core_cycle_counts': {      # Per-core core cycle counter deltas
-        'bmark:VM:variant': [
-            [                   # One list per process execution
-                [...], ...      # One list per core
-            ]
-    },
-    'aperf_counts': {...}       # Per-core APERF deltas
-                                # (structure same as 'core_cycle_counts')
-    'mperf_counts': {...}       # Per-core MPERF deltas
-                                # (structure same as 'core_cycle_counts')
-    'eta_estimates': {u"bmark:VM:variant": [t_0, t_1, ...], ...} # A dict mapping
-                  # benchmark keys to rough process execution times. Used internally,
-                  # users can ignore this.
-}
-```
-
-Some options exist to help inspect the results file:
-
-  * `--dump-reboots`
-  * `--dump-etas`
-  * `--dump-config`
-  * `--dump-audits`
-  * `--dump-temps`
-  * `--dump-data`
-
-```bash
-$ python krun.py --dump-config examples/example_results.json.bz2
-INFO:root:Krun starting...
-[2015-11-02 14:23:31: INFO] Krun starting...
-import os
-from krun.vm_defs import (PythonVMDef, NativeVMDef)
-from krun import EntryPoint
-
-# Who to mail
-MAIL_TO = []
-...
-
-$ python krun.py --dump-audit examples/example_results.json.bz2
-{
-    "cpuinfo":  "processor\t: 0\nvendor_id\t: GenuineIntel\ncpu family\t:
-...
-
-$ python krun.py --dump-reboots examples/example_results.json.bz2
-[2015-11-06 13:14:35: INFO] Krun starting...
-8
 ```
 
-## Testing your configurations
+Note, this will only work if you have followed the earlier steps to compile
+Krun with Java support.
 
-It is often useful to test a configuration file, without actually
-running a full benchmark (especially if the benchmark program is
-long). The best to test a config is to do something like:
 
-```bash
-$ ../krun.py --dry-run --quick --debug=INFO example.krun
-```
+### The Krun user
 
-See the "Development and Debug Switches" section for a description of these
-switches.
+Krun runs benchmarks under a new Unix user `krun`, which is wiped and re-added
+before every experiment. Your Krun build and your experiment must both be
+readable by the `krun` user for the experiment to run.
 
-Another switch, `--info`, reports various statistics about the setup described in the
-specified config file, such as the total number of process executions and which
-benchmark keys will be skipped etc.
 
 ## Creating your own experiments
 
-The configuration file `examples/example.krun` controls the experiment here.
-To create your own experiments, you can start by expanding on this example.
-The directory structure for the
+It is easiest to use `examples/example.krun` as a template for your own,
+new, experiments. The directory structure \laurie{"must be"?}:
```

Comment:
* Benchmarks must be in a `benchmarks` subdir.

https://github.com/softdevteam/krun/blob/cf1ee595c2b07e19790f750f9a650422a815e0ec/krun/vm_defs.py#L59

 * Each benchmark then has a subdir (to match the name of the benchmark in the config file)

 * Each benchmark variant then optionally has a subdirectory (we use on variant per language for our experiments):

https://github.com/softdevteam/krun/blob/cf1ee595c2b07e19790f750f9a650422a815e0ec/krun/vm_defs.py#L121-L132

The name of the benchmark file is also determined by the variant's entry point.